.vscode/settings.json

```diff
@@ -3,6 +3,7 @@
 	"editor.defaultFormatter": "esbenp.prettier-vscode",
 	"editor.formatOnSave": true,
 	"editor.rulers": [80],
+	"eslint.experimental.useFlatConfig": false,
```

Comment:
Can you separate this out? If the VS Code settings should be manually specifying this (though, why?) then I'd want to add this line to https://github.com/JoshuaKGoldberg/create-typescript-app.